src/core/env/Environment.ts

```diff
@@ -24,6 +24,7 @@ export type EnvVars = {
   INSTANCE_OWNER_HREF?: string;
   INSTANCE_OWNER_NAME?: string;
   MUIX_LICENSE_KEY?: string;
+  TILESERVER?: string;
```

Comment:
With the fallback, it should be possible to make this required.

```suggestion
  TILESERVER: string;
```

Reply:
I looked into this more closely, and this change won't work. So I recommend adding a fallback but not making this change to the type.